Example/ConsentViewController_ExampleTests/SPClientCoordinator/SPClientCoordinatorSpec.swift

```diff
@@ -461,6 +461,38 @@ class SPClientCoordinatorSpec: QuickSpec {
                     }
                 }
             }
+
+            describe("and expiration date is greater than current date") {
+                it("erases consent data and returns a message") {
+                    SPConsentManager.clearAllData()
```

Comment:
Do you have only one method for deleting the consent? 
What’s the behaviour is for instance you have a list of 2 campaigns and one of them is expiring while the other not yet?

Reply:
@carmelo-iriti the `.clearAllData` above is not related to consent expiration, but rather to clean all data before the test starts. 

The actual expiration logic is in the following lines: https://github.com/SourcePointUSA/ios-cmp-app/pull/505/files#diff-fc656d458a68144d63f7a40d9583ccf10a64bb53fd3cc75d0a30e13e8bca4a18R293

Keep in mind this is for GDPR only. For CCPA I implemented on #506